modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/20_validation.yml

```diff
@@ -94,28 +94,6 @@
         body:
           conflicts: cat
 
----
-"specifying size fails":
-  - requires:
-      cluster_features: ["gte_v8.0.0"]
-      reason: "size supported until 8"
-
-  - do:
-      index:
-        index:  test
-        id:     "1"
-        body:   { "text": "test" }
-
-  - do:
-      catch: /invalid parameter \[size\], use \[max_docs\] instead/
```

Comment:
Could we instead keep the test but generalize this regex to handle both the old specialized message and the new general one?

Reply:
Thanks, I've updated it!